ci-operator/step-registry/firewatch/report-issues/firewatch-report-issues-ref.yaml

```diff
@@ -27,6 +30,21 @@ ref:
     documentation: The Jira project you'd like bugs to be filed to if no rule matches the failure found
   - name: FIREWATCH_CONFIG
     documentation: The firewatch rules to be used. See https://github.com/CSPI-QE/firewatch for more information on how to define this.
+  - name: RH_PATTERNS_SERVER_API_TOKEN_PATH
```

Comment:
All of these new environment variables could be arguments for a `firewatch gitleaks` command with defaults set.